src/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/GhDampingFunction.hpp

```diff
@@ -0,0 +1,54 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines base-class GhDampingFunction.
+
+#pragma once
+
+#include <cstddef>
+#include <memory>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+/// Holds classes implementing GhDampingFunction (functions \f$R^n \to R\f$).
+namespace GeneralizedHarmonic::ConstraintDamping {
+template <size_t VolumeDim, typename Fr>
+class GaussianPlusConstant;
+
+/*!
+ * Encodes a function \f$R^n \to R\f$ where n is `VolumeDim` that represents
```

Comment:
There's no `brief` showing up in the documentation. Maybe just add a brief `\brief Base class defining interface for constraint damping functions.`?